yarn.lock

```diff
@@ -0,0 +1,1518 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+JSONSelect@0.4.0:
```

Comment:
Yarn locks the version of all dependencies, including subdependencies. This particular dependency comes from Browserify. Think of `yarn.lock` like `npm-shrinkwrap.json`.